ibtk/include/ibtk/SnapshotCache.h

```diff
@@ -0,0 +1,203 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2022 - 2022 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////////////// INCLUDE GUARD ////////////////////////////////
+
+#ifndef included_IBTK_SnapshotCache
+#define included_IBTK_SnapshotCache
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+
+#include <ibtk/config.h>
+
+#include "CellVariable.h"
+#include "EdgeVariable.h"
+#include "FaceVariable.h"
+#include "HierarchyDataOpsReal.h"
+#include "IntVector.h"
+#include "NodeVariable.h"
+#include "PatchHierarchy.h"
+#include "SideVariable.h"
+#include "tbox/Database.h"
+#include "tbox/Pointer.h"
+
+#include <map>
+#include <set>
+#include <string>
+#include <vector>
+
+/////////////////////////////// CLASS DEFINITION /////////////////////////////
+
+namespace IBTK
+{
+/*!
+ * \brief Class SnapshotCache provides a method of storing snapshots. This class can also be used to determine
+ * intermediate velocity fields when at a periodic steady state.
```

Comment:
```suggestion
 * intermediate velocity fields between snapshots. A periodic time interval can be specified for time-periodic cases.
```

Reply:
BTW, I don't see in the class definition where the periodic interval is specified.

Reply:
This class knows nothing of periodicity. Apparently I forgot to remove the comments about periodic intervals.